### PR TITLE
Ax 623

### PR DESCRIPTION
**Pull request by charlybongo**
# Description
//Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes Jira Ticket # 

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Is this a UI Change? If so please include screenshot of before and after states below

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have assigned 2 reviewers to check my work
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by charlybongo** [comment]
User is getting success popup after approve a Sell even if they don't have sufficient fund in wallet 

Status- fixed

File: lib/service/Dialog.dart
Comment (pearlson16) [inline lib/service/Dialog.dart:1082]: Can we rename this as well so its easier to understand what it does

**Review by pearlson16 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**
